### DISCUSSION TITLE
Why use such common ports?

### DISCUSSION THREAD

**Discussion by MrBreadWater** [SteamDeckHomebrew/decky-loader#420] [General]
8080 and 1337 are some of the most commonly used temporary ports. I have used them on my deck before. This clogs them up permanently? Why not just pick less frequently used ports? There shouldn't be any functional difference, afaik. Just a suggestion.

**Comment by PartyWumpus** [comment]
8080 is what valve uses for CEF debugging, so that isn't possible to change on decky's end.

8081 is used for the optional CEF forwarding which I think could be changed?

1337 is definitely a choice though, and could be changed.

Edit: looks like 8081 was also chosen by valve, as it's just what the `steam-web-debug-portforward` service uses.

**Reply by MrBreadWater** [reply]
Ah, fair enough on 8080 then. Tbf 1337 isnt extremely common either, I just happen to use it a lot haha. Alright I'm satisfied that it wasnt an arbitrary choice 😂
